ONSdigital/ssdc-rm-response-operations
# Add logging of remove member

# Motivation and Context
Log / audit when a user is removed

# What has changed
Logs nicely when a user is removed

# How to test?
In GCP, run this.  Add a user with supportTool.  Then remove them with ROPS.  Should log nicely

# Links
https://trello.com/c/omuCQhAk/3140-audit-on-permissions-5
